src/components/gltf-model-plus.js

```diff
@@ -362,7 +375,17 @@ AFRAME.registerComponent("gltf-model-plus", {
           attachTemplate(this.el, name, this.templates[name]);
         }
       }
+
       this.el.setObject3D("mesh", object3DToSet);
+
+      if (model.animations.length > 0) {
+        this.el.setAttribute("animation-mixer", "");
+
+        if (!this.el.querySelector("[loop-animation]")) {
+          this.el.setAttribute("loop-animation", "");
```

Comment:
Does this allow us to remove the animation mixer in `media-loader` that plays the loading cube's animation? 
https://github.com/mozilla/hubs/blob/master/src/components/media-loader.js#L74

Reply:
It would, yeah. Whether that should be the responsibility of `gltf-model-plus` or if we should be adding those components in `media-loader` or somewhere else, I don't know.